docs/source/releases/lts/103_0.rst

```diff
@@ -0,0 +1,602 @@
+  ansible’s `user
+  <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html>`__
+  module.  Likewise, if a test needs a service installed or up and
+  running: instead of writing all that code, the test writer can
+  describe that dependency using the `service
+  <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html>`__
+  ansible module.
+
+* By setting the ``spawner.podman.avocado_spawner_egg`` configuration
+  users can now control the exact Avocado package that will be
+  automatically deployed within Podman containers.
+
+* When using the podman spawner, if the test URI looks like a
+  filesystem path, it will be exposed read only to the container. This
+  makes running Avocado tests inside containers transparent in many
+  cases.
+
+* A Podman image can now be treated as a dependency. This is currently
+  mostly useful to tests that will manually create containers, but
+  it’s expected to also be leveraged by the podman spawner and suite
+  and job level dependencies in the near future.
+
+* :class:`avocado.core.nrunner.runnable.Runnable` instances will now,
+  under some circumstances, automatically include the current
+  environment’s relevant and required configuration. This makes
+  standalone executions of ``avocado-runner-*`` commands easier, while
+  previously it would require manually entering all required
+  configuration.
+
+* The ``--failfast`` feature now acts on any :data:`"not OK"
+  <avocado.core.teststatus.STATUSES_NOT_OK>` result.  Previously, it
+  would only act (that is, abort the job) upon a ``FAIL`` test result.
+
+* Test writers can now access a test’s status while the
+  :meth:`avocado.core.test.Test.tearDown` method is being run.
+
+* Support was added for ``Coverage.py`` when running
+  ``avocado-instrumented`` tests (currently limited to the
+  process spawner).
+
+* A results plugin for `Beaker <https://beaker-project.org>`_ is
+  now available and works out of the box without any manual
+  configuration needed.
+
+* The task identifiers can now be configured as a format string that
+  utilizes the attributes of the runnable. This allows users to define
+  how test results will be named and presented.
+
+* When using the Job API, test suites can be enabled or
+  disabled. Having a disabled test suite means it won’t be
+  executed. This eases the creation of custom jobs where the user can
+  choose to run a subset of suites.
+
+* The remote spawner is a new optional plugin that makes use of remote
+  `aexpect <https://github.com/avocado-framework/aexpect/>`__ sessions
+  (to remote hosts or equivalently remote containers behind remote hosts
+  forwarded via specific ports) as slots to schedule test runs on.
+
+* The format of test log messages have been reverted to more closely
+  match previous LTS (92.x) format like line number and Python module
+  name (see `#5721
+  <https://github.com/avocado-framework/avocado/issues/5721>`__).
```

Comment:
I would remove the issue links from all notes. I like those links and I think we should have them in the all new releases (like in 103). But I think we should remove them from here, because we don't have them in other notes and I would like to keep the style same.